aito/cli/sub_commands/database_sub_command.py

```diff
@@ -92,12 +159,77 @@ def parse_and_execute(self, parsed_args: Dict):
         return 0
 
 
+class CopyTableSubCommand(SubCommand):
+    def __init__(self):
+        super().__init__('copy-table', 'copy a table')
+
+    def build_parser(self, parser):
+        parser.add_aito_default_credentials_arguments()
+        parser.add_argument('table-name', type=str, help="the name of the table to be copied")
+        parser.add_argument('copy-table-name', type=str, help="the name of the new copy table")
+        parser.add_argument(
+            '--replace', action='store_true',
+            help="replace an existing table of which name is the name of the copy table"
```

Comment:
I'm not sure if I understand correctly what replace does, is it this?

```suggestion
            help="replace the data of an existing table with the data of the table to be copied"
```